go.mod

```diff
@@ -5,7 +5,7 @@ go 1.22.0
 toolchain go1.22.7
 
 require (
-	github.com/Layr-Labs/eigenda v0.8.6-rc.0.0.20250130173836-0d293cc03198
+	github.com/Layr-Labs/eigenda v0.9.0-rc.0.0.20250228180652-c7f0c3d62bbf
```

Comment:
TODO: make an eigenda release and update this before we merge

Reply:
Update: eigenda release isn't going to happen in the immediate term. This PR will merge with this direct commit reference, and we will just need to coordinate the next proxy release to wait at least until the next eigenda release is complete